langchain-core/src/utils/json_schema.ts

```diff
@@ -22,16 +20,8 @@ export { deepCompareStrict, Validator } from "@cfworker/json-schema";
  */
 export function toJsonSchema(schema: InteropZodType | JSONSchema): JSONSchema {
   if (isZodSchemaV4(schema)) {
-    const inputSchema = interopZodTransformInputSchema(schema, true);
-    if (isZodObjectV4(inputSchema)) {
-      const strictSchema = interopZodObjectStrict(
-        inputSchema,
-        true
-      ) as ZodObjectV4;
-      return toJSONSchema(strictSchema);
-    } else {
-      return toJSONSchema(schema);
```

Comment:
Am I correct to assume that `toJSONSchema(schema)` was a typo and that it should have been `toJSONSchema(inputSchema)`?

If not, please let me know, as this PR effectively makes that change.